src/views/FAQPage/FAQItem.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <Fragment>
+    <div class="faq-question" @click="() => toggleAccordion(question)">{{ question }}</div>
+    <div class="faq-answer">
+      <div class="answer">{{ answer }}</div>
+    </div>
+  </Fragment>
+</template>
+<script>
+import { Fragment } from 'vue-fragment';
+
+const FAQItem = {
+  name: 'faqitem',
+  components: {
+    Fragment,
+  },
+
+  props: ['question', 'answer', 'toggleAccordion'],
```

Comment:
props should also resolve first if possible

Reply:
Not sure what you mean. Could you please clarify a bit? Or maybe an example would help.